src/include/connection.h

```diff
@@ -400,6 +400,7 @@ struct __wt_connection_impl {
     uint64_t sweep_handles_min;     /* Handle sweep minimum open */
 
     /* Set of btree IDs not being rolled back */
+    bool rollback_to_stable;
```

Comment:
I would have expected this as a session flag rather than connection-wide.  Eviction runs during rollback-to-stable, and setting this as written will change the behaviour of all reconciliation that runs  concurrent with rollback-to-stable.  Is that what you want?  Either way, it needs a big comment explaining what it's doing.

Reply:
I added a comment to explain the need for it. Can you please review it again.

Reply:
Followed up with @kommiharibabu in person, write up here: https://jira.mongodb.org/browse/WT-5421?focusedCommentId=2799131&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2799131

Reply:
Thanks @michaelcahill for the discussion. I fixed it using the `__wt_update` flag to indicate the update is restored from history store for rollback to stable operation.